resources/views/Staff/poll/edit.blade.php

```diff
@@ -0,0 +1,47 @@
+@extends('layout.default')
+
+@section('title')
+    <title>Polls - {{ config('other.title') }}</title>
+@endsection
+
+@section('breadcrumb')
+    <li>
+        <a href="{{ route('staff.dashboard.index') }}" itemprop="url" class="l-breadcrumb-item-link">
+            <span itemprop="title" class="l-breadcrumb-item-link-title">@lang('staff.staff-dashboard')</span>
+        </a>
+    </li>
+    <li>
+        <a href="{{ route('staff.polls.index') }}" itemprop="url" class="l-breadcrumb-item-link">
+            <span itemprop="title" class="l-breadcrumb-item-link-title">@lang('poll.polls')</span>
+        </a>
+    </li>
+    <li>
+        <a href="{{ route('staff.polls.edit', ['id'=> $poll->id]) }}" itemprop="url" class="l-breadcrumb-item-link">
+            <span itemprop="title" class="l-breadcrumb-item-link-title">@lang('poll.edit-poll') {{$poll->slug}}</span>
+        </a>
+    </li>
+@endsection
+
+@section('content')
+    <div class="container">
+        <div class="row">
+            <div class="page-title">
+                <h1>@lang('poll.edit-poll')</h1>
+            </div>
+            <div class="col-md-12">
+                <div class="panel panel-chat">
+                    <div class="panel-heading">
+                        @lang('common.edit')
```

Comment:
Oh these 3 lines should be `@lang('poll-edit')`.  I'm localizing the Staff/* views. @lang(trans_choice('common.a-an-art',false)) is used for switching a/an in English.

Reply:
2643fa6f8714e732390668e74b8754b537e384c6